.github/workflows/msvc-build.yml

```
@@ -0,0 +1,113 @@
+jobs:
+  vs2022-build:
+    name: MSVC 2022 Build (${{ matrix.platform }}-Release)
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        platform: [x64, Win32]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)